File: server/MWL.ContentService/MWL.ContentService.Domain/Entities/Review.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MWL.ContentService.Domain.Entities
{
    public class Review
    {
        public int Id { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime CreatedAt { get; set; }
        public int MovieId { get; set; }
        public string UserId { get; set; }
    }
}

File: client/src/lib/api/review.js
import base from './base';

const postReview = async ({
    userId,
    movieId,
    rating,
    comment
}) => {
    return await base.post('/review', {
        "userId": userId,
        "movieId": movieId,
        "rating": rating,
        "comment": comment
    });
}

export default {
    postReview
}